pkgs/tools/misc/vdirsyncer/default.nix

```diff
@@ -35,6 +35,13 @@ in pythonPackages.buildPythonApplication rec {
 
   checkInputs = with pythonPackages; [ hypothesis pytest pytest-localserver pytest-subtesthack ] ++ [ glibcLocales ];
 
+  patches = [
+    (fetchpatch {
+      url = https://github.com/pimutils/vdirsyncer/pull/779.patch;
```

Comment:
This link is not stable. If you make changes to the PR, the hash will change. Instead you can pick the patches (https://github.com/pimutils/vdirsyncer/pull/779/commits/29417235321c249c65904bc7948b066ef5683aee, https://github.com/pimutils/vdirsyncer/pull/779/commits/22ad88a6b18b0979c5d1f1d610c1d2f8f87f4b89) individually.